adapter/validator/src/test/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidatorTest.java

```diff
@@ -1141,4 +1141,51 @@ void emptyRequiredValueGenerateError() {
         assertEquals("Missing value for field:type_date marked as required in entity with id:test_id",
                 notice.getDescription());
     }
+
+    @Test
+    void checkCurrencyCodeValidation() {
```

Comment:
After discussion, not sure we gain in clarity. Please refactor to seperate the two tests and give them specific methods names